From 7abf0cd119750820332f58413f56d0e7bba7f925 Mon Sep 17 00:00:00 2001
From: sahbot <support.opensource@softathome.com>
Date: Thu, 16 Feb 2023 08:58:09 +0100
Subject: [PATCH] Issue: HOP-2538 [SSH][AMX] Processes spawned by dropbear
 instance managed SSH Manager ignores SIGINT [fix]

GitOrigin-RevId: 15ec2c3a02cde3851b0e82d0670e59c75f0b220a
---
 src/amxp_subproc.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/amxp_subproc.c b/src/amxp_subproc.c
index 423112d..ed8246d 100644
--- a/src/amxp_subproc.c
+++ b/src/amxp_subproc.c
@@ -166,10 +166,6 @@ static void amxp_subproc_exec_child(amxp_subproc_t* const subproc,
     pthread_sigmask(SIG_SETMASK, NULL, &sig_set);
     pthread_sigmask(SIG_UNBLOCK, &sig_set, NULL);
 
-    sigemptyset(&sig_set);
-    sigaddset(&sig_set, SIGINT);
-    pthread_sigmask(SIG_BLOCK, &sig_set, NULL);
-
     execvp(argv[0], argv);
 
     _exit(EXIT_FAILURE);
